src/main/java/frc/robot/subsystems/Hood.java

```diff
@@ -42,4 +44,8 @@ public void periodic() {
     SmartDashboard.putNumber("Hood Motor", getHoodMotorEncoderCount());
 
   }
+
+  public void setAngle(double p_angle) {
+    hoodMotor.set(ControlMode.PercentOutput, p_angle);
```

Comment:
Remember we had safety sensors. Please include them to keep the robot from killing itself.